pkg/workload/workloadslicing/workloadslicing.go

```diff
@@ -0,0 +1,311 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package workloadslicing contains definitions to support workload slicing functionality.
+package workloadslicing
```

Comment:
I would like to move this package level up (not under workload). The reason is that many of the functions here actually are called from Jobframework packages, and are not directly related to workloads.

Reply:
thx